client/i18n/locales/chinese-traditional/translations.json

```
@@ -364,6 +364,7 @@
   },
   "misc": {
     "offline": "你已離線，學習進度可能不會被保存",
+    "server-offline": "The server could not be reached and your progress may not be saved. Please contact support if this message persists",
```

```suggestion
    "server-offline": "The server could not be reached and your progress may not be saved. Please contact support@freeCodeCamp.org if this message persists",
```
People may not know how to contact support, so the email address could be helpful.

Don't worry about the changing other languages, Crowdin will take care of that.

---

I am little worried about the message length. It's already too long for my liking and may overflow on smaller screens if it's even longer. Perhaps I could make the suppor a clickable `mailto:` link ?

---

Sounds good to me!